lib/theme-options/site-options.php

```diff
@@ -24,7 +24,7 @@
       'name'    => __( 'Evento Actual/Siguente', 'IGV' ),
       'id'      => $prefix . 'radio_evento_current',
       'type'    => 'select',
-      'show_option_none' => false,
+      'show_option_none' => true,
```

Comment:
I don't understand the bug. Plz open a pull request. Also check the local timezone on your local server.